blocks/footer/footer.css

```diff
@@ -1,14 +1,173 @@
 footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+  background-color: #222;
+  color: #fff;
```

Comment:
Can you use the variable here also in other places in this file ?

Reply:
Done